ext/XS-APItest/t/boolean-thr.t

```diff
@@ -0,0 +1,38 @@
+#!./perl
+
+use strict;
+use warnings;
+
+use Config ();
+use if !$Config::Config{usethreads}, 'Test::More',
+    skip_all => "perl does not support threads";
```

Comment:
perl does support threads.  *This* perl does not

Reply:
Fiiiiine

[test-SvIsBOOL-using-XS-APItest 1aae98d682] Wording fix in test skip message